app/services/data_migrations/remove_application_choices_in_the_incorrect_cycle.rb

```diff
@@ -0,0 +1,18 @@
+module DataMigrations
+  class RemoveApplicationChoicesInTheIncorrectCycle
+    TIMESTAMP = 20210609084557
+    MANUAL_RUN = true
+
+    def change
+      ApplicationChoice
+      .joins(:application_form, course_option: [:course])
```

Comment:
So in the previous PR you mentioned using current_course_option.

I ran a query and there are two. I'd like to investigate those separately as they really shouldn't have been able to get into that state through the change offer flow.

I've updated the invariant check to use current_course_option instead though!